plugins/modules/vm_list_group_by_clusters_info.py

```diff
@@ -191,7 +191,12 @@ def main():
 
     vmware_vm_list_group_by_clusters_mgr = VmwareVMList(module)
     vm_list_group_by_clusters_info = vmware_vm_list_group_by_clusters_mgr.get_vm_list_group_by_clusters()
-    module.exit_json(changed=False, vm_list_group_by_clusters_info=vm_list_group_by_clusters_info)
+    # Till we will release the next major version 2.0.0 we should keep the deprecated module return value
+    if not module._name.endswith('_info'):
+        module.exit_json(changed=False, vm_list_group_by_clusters_info=vm_list_group_by_clusters_info)
```

Comment:
If I am not mistaken, this should be:
```
module.exit_json(changed=False, vm_list_group_by_clusters=vm_list_group_by_clusters_info
```

Reply:
Fixed it
thanks

Reply:
Looks still the same for me

Reply:
Ah I see, you remove the `not`.

Reply:
Actually, it's imho wrong. Because what you want is to keep backward compatibility. That means, you want to return same value for `_info` + old value. But for old module, you just keep returning what you did.

Reply:
OK I changed it